src/core/timer.h

```diff
@@ -41,6 +41,7 @@ struct Timer {
 
         usec_t accuracy_usec;
         usec_t random_usec;
+        usec_t random_value_usec;
```

Comment:
this should be initialized in timer_init() to USEC_INFINITY too. In case someone turns on the option in a unit file while the timer is already up and issues `systemctl daemon-reload`(and thus we don't run through timer_start() again) we'd otherwise calculate with random_value_usec=0 here...